### ISSUE TITLE
Incorrect edges count after committing with sbtree-based ridbag

### ISSUE THREAD

**Issue by johny65** [orientechnologies/orientdb#9805]
## Expected behavior  

A committed vertex should return the correct edges after adding a new edge.

## Actual behavior 

When sbtree-based rigbad is activated (server configured with `ORIENTDB_SETTINGS="-DridBag.embeddedToSbtreeBonsaiThreshold=-1"`), adding an edge to a vertex and then committing, the vertex returns incorrect count of edges.

This is the same as #9076 but for OrientDB version 3.2 with Multi Model API. The behavior is correct in version 3.1. As noted in <https://github.com/orientechnologies/orientdb/issues/9076#issuecomment-593975806>, in 3.0 configuring the client with `GlobalConfiguration.RID_BAG_EMBEDDED_TO_SBTREEBONSAI_THRESHOLD.setValue(-1)` fixes the case, but this is not true in 3.2.

## Steps to reproduce  

```java
        ODatabaseSession g = pool.acquire();
        g.begin();
        OVertex vertex1 = g.newVertex("V"); vertex1.save();
        OVertex vertex2 = g.newVertex("V"); vertex2.save();
        g.commit();
        
        int version = vertex1.getProperty("@version");
        System.out.println("Version: " + version);
        System.out.println("vertex1 out: " + countEdges(vertex1, ODirection.OUT));
        System.out.println("vertex2 in: " + countEdges(vertex2, ODirection.IN));
        /* output:
           Version: 1
           vertex1 out: 0
           vertex2 in: 0
        */
        
        g.begin();
        vertex1.addEdge(vertex2);
        vertex1.save();
        
        version = vertex1.getProperty("@version");
        System.out.println("Pre-commit:");
        System.out.println("Version: " + version);
        System.out.println("vertex1 out: " + countEdges(vertex1, ODirection.OUT));
        System.out.println("vertex2 in: " + countEdges(vertex2, ODirection.IN));
        /* output:
           Pre-commit:
           Version: 1
           vertex1 out: 1
           vertex2 in: 1
        */
        
        g.commit();
        
        version = vertex1.getProperty("@version");
        System.out.println("Post-commit:");
        System.out.println("Version: " + version);
        System.out.println("vertex1 out: " + countEdges(vertex1, ODirection.OUT));
        System.out.println("vertex2 in: " + countEdges(vertex2, ODirection.IN));
        /* output:
           Post-commit:
           Version: 2
           vertex1 out: 0 <- INCORRECT
           vertex2 in: 0 <- INCORRECT
        */
        
        g.close();
        
        g = pool.acquire();
        vertex1 = g.load(vertex1.getIdentity());
        vertex2 = g.load(vertex2.getIdentity());

        version = vertex1.getProperty("@version");        
        System.out.println("Reload in new transaction:");
        System.out.println("Version: " + version);
        System.out.println("vertex1 out: " + countEdges(vertex1, ODirection.OUT));
        System.out.println("vertex2 in: " + countEdges(vertex2, ODirection.IN));
        /* output:
           Reload in new transaction:
           Version: 2
           vertex1 out: 1
           vertex2 in: 1
        */
```

```java
    private int countEdges(OVertex v, ODirection dir) throws Exception {
        int c = 0;
        Iterator it = v.getEdges(dir).iterator();
        while (it.hasNext()) { c++; it.next(); }
        return c;
    }
```

**Comment by tglman** [comment]
Hi,

Tried to write a test out of the specified case, but seems to work fine, could you help me to reproduce?
here is the commit: https://github.com/orientechnologies/orientdb/commit/effc3138befc2c60bf4f46d6fcd04b4c9737da2c

Regards

**Comment by johny65** [comment]
Hi @tglman. The server must be configured with `ORIENTDB_SETTINGS="-DridBag.embeddedToSbtreeBonsaiThreshold=-1"` (I edit the server.sh file for convenience). Changing the option in the client side does not take effect.
Can you try with this configuration?

**Comment by tglman** [comment]
Hi,

I could reproduce the case in remote and fix it, the fix will be released with the next hotfix 3.2.18.

Regards

**Comment by johny65** [comment]
Hi, I tested 3.2.18 and it is fixed. Thanks.
